samples/net/http_server/prj_netusb.conf

```diff
@@ -0,0 +1,59 @@
+CONFIG_NETWORKING=y
+CONFIG_NET_TCP=y
+CONFIG_RANDOM_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_NET_LOG=y
+CONFIG_INIT_STACKS=y
+
+CONFIG_NET_PKT_RX_COUNT=16
+CONFIG_NET_PKT_TX_COUNT=16
+CONFIG_NET_BUF_RX_COUNT=16
+CONFIG_NET_BUF_TX_COUNT=16
+
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=2
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=4
+
+CONFIG_NET_MAX_CONTEXTS=16
+
+CONFIG_SYS_LOG_SHOW_COLOR=y
+CONFIG_SYS_LOG_NET_LEVEL=2
+CONFIG_NET_DEBUG_HTTP=y
+
+CONFIG_HTTP=y
+CONFIG_HTTP_SERVER=y
+
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+#CONFIG_NET_DHCPV4=y
+
+# Disable HTTPS because of low memory
+#CONFIG_HTTPS=y
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_BUILTIN=y
+CONFIG_MBEDTLS_CFG_FILE="config-mini-tls1_2.h"
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=12000
+
+CONFIG_NET_APP_SETTINGS=y
+CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
+
+CONFIG_NET_APP_NEED_IPV6=y
+CONFIG_NET_APP_NEED_IPV4=y
+
+CONFIG_NET_SHELL=y
+CONFIG_NET_STATISTICS=y
+
+CONFIG_NET_MGMT=y
+CONFIG_NET_MGMT_EVENT=y
+
+CONFIG_RAM_SIZE=280
```

Comment:
why do you tweak this system info?

Reply:
It's not me, this is the diff:

21a22
> CONFIG_HTTP=y
28c29,30
< CONFIG_HTTPS=y
---
> # Disable HTTPS because of low memory
> #CONFIG_HTTPS=y
48a51,59
> 
> # USB Device settings
> CONFIG_USB=y
> CONFIG_USB_DEVICE_STACK=y
> CONFIG_SYS_LOG_USB_DRIVER_LEVEL=3
> CONFIG_SYS_LOG_USB_LEVEL=3
> 
> # Select USB Configurations
> CONFIG_USB_DEVICE_NETWORK_ECM=y

Reply:
Try without.